client/my-sites/checkout/checkout-thank-you/redesign-v2/masterbar-styled/default-contact.tsx

```diff
@@ -48,15 +47,14 @@ const ContactContainer = styled.div`
 `;
 
 export function DefaultMasterbarContact() {
-	const translate = useTranslate();
 	const { helpCenterButtonCopy, helpCenterButtonLink, toggleHelpCenter } = useCheckoutHelpCenter();
 
 	return (
 		<ContactContainer>
-			<label>{ helpCenterButtonCopy ?? translate( 'Need extra help?' ) }</label>
+			{ helpCenterButtonCopy && <label>{ helpCenterButtonCopy }</label> }
```

Comment:
Do we need to check the `helpCenterButtonCopy` here before printing the label? If so, why don't we do the same thing for the `helpCenterButtonLink`.
From what I can see, both might not exist.

Reply:
Because `helpCenterButtonCopy` is not always returned by the `useProductsWithPremiumSupport` util – it's only set on the DIFM flow. `helpCenterButtonLink` is always guaranteed to exist.

I added some types in the last commit to enforce it.

Reply:
That's `trunk`. It's guaranteed to exist in this branch. See https://github.com/Automattic/wp-calypso/blob/cb18d24655023e4c7210332212e126bd4981925b/packages/help-center/src/hooks/use-products-with-premium-support.ts#L58-L67